k8s/aat/common-overlay/rd/kustomization.yaml

```diff
@@ -6,5 +6,6 @@ bases:
 - ../../../namespaces/rd/rd-caseworker-ref-api/rd-caseworker-ref-api.yaml
 - ../../../namespaces/rd/rd-judicial-api/rd-judicial-api.yaml
 - ../../../namespaces/rd/rd-user-profile-api/rd-user-profile-api.yaml
+- ../../../namespaces/rd/rd-professional-api/rd-professional-api.yaml
 patchesStrategicMerge:
-- ../../../namespaces/rd/rd-user-profile-api/aat.yaml
+- ../../../namespaces/rd/rd-user-profile-api/aat.yaml
```

Comment:
i think you can remove patchesStrategicMerge from kustomizaton yaml since  env specific files are removed

Reply:
It was not added in this PR. We have to make a change to this repo as well. Will do this later